### PR TITLE
Issue #53 fix: Allowed count config is not working as per the requirement

### PR DESCRIPTION
**Pull request by vaibhavsTekdi**
**Steps to reproduce:**

- Set the Allowed Count as 1 for a particular UCM type from admin panel
- Open this UCM form in two different browser tabs.
- Submit the form from two different browser tabs.

Check the data against this UCM type in the database or on respective UCM list, you will save the multiple records even if this UCM type have only one time for submission provision.

### CODE REVIEW

File: src/components/com_tjucm/site/controllers/itemform.php [hunk 1]
```diff
@@ -212,6 +212,33 @@ public function save($key = null, $urlVar = null)
...
+		$typeData = $tjUcmModelType->getItem($typeId);
+		$allowedCount = $typeData->allowed_count;
+
+		// 0 : add unlimited records against this UCM type
+		$allowedCount = empty($allowedCount) ? 0 : $allowedCount;
+
+		$user   = JFactory::getUser();
+		$userId = $user->id;
+
+		$allowedToAdd = $model->allowedToAddTypeData($userId, $this->client, $allowedCount);
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/site/controllers/itemform.php]: @vaibhavsTekdi 
such validations should be added in model

Comment (vaibhavsTekdi) [reply src/components/com_tjucm/site/controllers/itemform.php]: done the changes

File: src/components/com_tjucm/site/models/itemform.php [hunk 2]
```diff
@@ -55,6 +55,8 @@ public function __construct($config = array())
 	{
 		$this->common  = new TjucmFunList;
 
+		$this->isajax = ($_SERVER['HTTP_X_REQUESTED_WITH'] == 'XMLHttpRequest') ? true : false;
```
Comment (manojLondhe) [inline src/components/com_tjucm/site/models/itemform.php]: see if you can use this instead  to access $_SERVER  values https://docs.joomla.org/Retrieving_request_data_using_JInput#Getting_Values_from_a_Specific_Super_Global

Comment (manojLondhe) [reply src/components/com_tjucm/site/models/itemform.php]: @vaibhavsTekdi Fix scrutinizer issues

Comment (vaibhavsTekdi) [reply src/components/com_tjucm/site/models/itemform.php]: > see if you can use this instead to access $_SERVER values https://docs.joomla.org/Retrieving_request_data_using_JInput#Getting_Values_from_a_Specific_Super_Global

used the value from Super Global...

Comment (vaibhavsTekdi) [reply src/components/com_tjucm/site/models/itemform.php]: > @vaibhavsTekdi Fix scrutinizer issues
done..

**Pull request merged**